File: 2849. Determine if a Cell Is Reachable at a Given Time/Determine if a Cell Is Reachable at a Given Time.cpp
class Solution {
public:
    bool isReachableAtTime(int sx, int sy, int fx, int fy, int t) {
        if ((sx == fx) && (sy == fy) && (t == 1)){
            return false;
        }
        int XD = abs(sx - fx);
        int YD = abs(sy - fy);
        if ((XD <= t) && (YD <= t)){
            return true;
        }
        return false;
    }
};

File: 2849. Determine if a Cell Is Reachable at a Given Time/Determine if a Cell Is Reachable at a Given Time.py
class Solution:
    def isReachableAtTime(self, sx: int, sy: int, fx: int, fy: int, t: int) -> bool:
        if ((sx == fx) and (sy == fy) and (t == 1)):
            return False
        XD = abs(sx - fx)
        YD = abs(sy - fy)
        if (max(XD,YD,t) <= t):
            return True
        return False
